met-web/src/components/engagement/old-view/widgets/Map/MapWidget.tsx

```diff
@@ -56,76 +60,92 @@ const MapWidget = ({ widget }: MapWidgetProps) => {
-                    </Grid>
-                    <Grid item xs={12}>
-                        <Box
-                            ref={mapContainerRef}
-                            sx={{
-                                width: '100%',
-                                height: '370px',
-                            }}
-                        >
-                            <MetMap
-                                geojson={geoJSONDecode(map.geojson)}
-                                longitude={map.longitude}
-                                latitude={map.latitude}
-                                markerLabel={map.marker_label}
-                                zoom={calculateZoomLevel(mapWidth, mapHeight, geoJSONDecode(map.geojson))}
-                            />
-                        </Box>
-                    </Grid>
-                    <When condition={isLargeScreen}>
-                        <Grid container item xs={12} alignItems={'center'} justifyContent={'flex-start'}>
-                            <Grid item paddingRight={'5px'}>
-                                <IconButton icon={faExpand} onClick={() => setOpen(true)} backgroundColor="none" />
-                            </Grid>
-                            <Grid item>
-                                <Link onClick={() => setOpen(true)} tabIndex={0} onKeyPress={() => setOpen(true)}>
-                                    View Expanded Map
-                                </Link>
-                            </Grid>
-                        </Grid>
-                        <ExpandModal map={map} open={open} setOpen={setOpen} />
-                    </When>
+        <Grid container justifyContent={{ xs: 'center' }} alignItems="center" rowSpacing={2}>
+            <Grid
+                item
+                container
+                justifyContent={{ xs: 'center', md: 'flex-start' }}
+                flexDirection={'column'}
+                xs={12}
+                paddingBottom={0}
+            >
+                <MetHeader3 style={metHeader3Styles}>{widget.title}</MetHeader3>
+            </Grid>
+            <Grid item xs={12}>
+                <Box sx={outerContainerStyles}>
+                    <Box sx={innerContainerStyles}>
+                        <MetMap
+                            geojson={geoJSONDecode(map.geojson)}
+                            longitude={map.longitude}
+                            latitude={map.latitude}
+                            markerLabel={map.marker_label}
+                            zoom={calculateZoomLevel(mapWidth, mapHeight, geoJSONDecode(map.geojson))}
+                        />
+                    </Box>
+                </Box>
+            </Grid>
+            <When condition={isLargeScreen}>
+                <Grid container item xs={12} alignItems={'center'} justifyContent={'flex-start'}>
+                    <Link onClick={() => setOpen(true)} sx={linkStyles} tabIndex={0} onKeyDown={() => setOpen(true)}>
```

Comment:
Although this was a link previously, I think it'd be better as a button. Users generally expect links to navigate them somewhere, whereas buttons are used for forms or interacting with the on-page UI.